core/scripts/bareos-regress.conf.in

```diff
@@ -1,3 +1,10 @@
+#
```

Comment:
Does the regression testing with this config work? If the regression tests do not work anymore we could also remove everything that is left from them also.

Reply:
I think, this file is still useful. I recommend to handle a regress specific cleanup in a separate PR.